backends/advanced/init.py

```
@@ -809,12 +785,16 @@ def setup_neo4j(self):
         neo4j_password = getattr(self.args, "neo4j_password", None)
 
         if neo4j_password:
-            self.console.print(f"[green]✅[/green] Neo4j: password configured via wizard")
+            self.console.print(
+                f"[green]✅[/green] Neo4j: password configured via wizard"
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)